test/cppsuite/configs/config_poc_test_insert_heavy.txt

```diff
@@ -0,0 +1,58 @@
+# Used as a stress test for the framework.
+duration_seconds=300,
+cache_size_mb=2000,
+statistics_config=
+(
+    type=all,
+    enable_logging=true
+),
+checkpoint_manager=
+(
+    enabled=true,
+    op_rate=20s
+),
+runtime_monitor=
+(
+    stat_cache_size=
+    (
+        enabled=true,
+        limit=100
+    )
+),
+timestamp_manager=
+(
+    enabled=true,
+    oldest_lag=30,
+    op_rate=1s,
+    stable_lag=10
+),
+workload_generator=
+(
+    populate_config=
+    (
+        collection_count=100,
+        key_count_per_collection=500,
+        key_size=100,
+        thread_count=20,
+        value_size=2000
+    ),
+    insert_config=
+    (
+        key_size=100,
+        op_rate=10ms,
+        ops_per_transaction=(max=100,min=0),
+        thread_count=50,
+        value_size=2000
+    ),
+    read_config=
+    (
+        op_rate=3ms,
+        ops_per_transaction=(max=100,min=50),
+        thread_count=10
+    ),
+),
+# Tracking is currently disabled as verification can't handle rollbacks.
```

Comment:
This should be removed as #6683 will be merged in beforehand

Reply:
As discussed with @tetsuo-cpp , it wont be fixed with #6683 but with WT-7636. You can leave this for now.